includes/rest-api/class-sensei-rest-api-course-users-controller.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * File contains the Sensei_REST_API_Course_Users_Controller class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Course users actions controller class.
+ *
+ * @since x.x.x
+ */
+class Sensei_REST_API_Course_Users_Controller extends \WP_REST_Controller {
+
+	/**
+	 * Routes namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace;
+
+	/**
+	 * Routes prefix.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'course-users/batch';
+
+	/**
+	 * Sensei_REST_API_Course_Structure_Controller constructor.
+	 *
+	 * @param string $namespace Routes namespace.
+	 */
+	public function __construct( $namespace ) {
+		$this->namespace = $namespace;
+	}
+
+	/**
+	 * Register the REST API endpoints for Course Structure.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			[
+				[
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => [ $this, 'add_users_to_courses' ],
```

Comment:
What do you think about standardizing the callback names?

For example, we could use `create_item`, `get_item`, `get_items`, `delete_item`, etc (this is a WP standard I think). For batch, maybe we could use `batch_create_items`, `batch_delete_items`?

Here are a few examples from woo: [batch](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-settings-controller.php#L33-L40), [the others](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/rest-api/Controllers/Version1/class-wc-rest-taxes-v1-controller.php).